swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Lets try to put this under the eng folder instead of the root directory.

Reply:
Same as above.